opm/common/TimingMacros.hpp

```diff
@@ -19,6 +19,12 @@
 #ifndef OPM_TIMINGMACROS_HPP
 #define OPM_TIMINGMACROS_HPP
 
+#if HAVE_TRACY
+#define TRACY_ENABLE 1
+#include <tracy/Tracy.hpp>
+#define OPM_TIMEBLOCK(blockname) ZoneNamedN(blockname, #blockname, true);
+#define OPM_TIMEBLOCK_LOCAL(blockname) ZoneNamedN(blockname, #blockname, true);
```

Comment:
i guess so. to avoid a lot of options, i think having the user edit the file to enable this is an acceptable compromise (defaults to 0).

Reply:
>> We might want to consider putting the `_LOCAL` macro behind one more level of guards.
>
> i guess so. to avoid a lot of options, i think having the user edit the file to enable this is an acceptable compromise (defaults to 0).

I agree.